### PR TITLE
Feature/import unlabeled dataset

### PR DESCRIPTION
**Pull request by cgill95**
Solving Issue #658 with this change.

Now it is possible to upload a CSV file that does not contain label for all text fields. Thus making it possible to upload partially labeled text together with labeled text.

### CODE REVIEW

File: app/api/utils.py [hunk 1]
```diff
@@ -385,10 +385,10 @@ def parse_excel_csv_reader(reader):
                 yield data
                 data = []
             # Only text column
-            if len(row) == len(columns) and len(row) == 1:
-                data.append({'text': row[0]})
+            if len(row) <= len(columns) and len(row) == 1:
+                data.append({'text': row[0] })
```
Comment (Hironsan) [inline app/api/utils.py:389]: Please remove an unnecessary space after `row[0]`.

File: app/api/utils.py [hunk 2]
```diff
@@ -385,10 +385,10 @@ def parse_excel_csv_reader(reader):
...
             # Text, labels and metadata columns
-            elif len(row) == len(columns) and len(row) >= 2:
+            elif len(row) <= len(columns) and len(row) >= 2:
```
Comment (Hironsan) [inline app/api/utils.py:391]: Simplify chained comparison:

```python
2 <= len(row) <= len(columns)
```

File: app/api/tests/data/example.csv [hunk 3]
```diff
@@ -1,4 +1,4 @@
-text,label
-AAA,Positive
-BBB,Positive
+text,label,meta
+AAA
+BBB,Positive,The following is meta data
```
Comment (Hironsan) [inline app/api/tests/data/example.csv:3]: I tested the case of `text` and `meta` but it failed.

```
text,label,meta
text1,,meta1
text2,,meta2
```

Comment (cgill95) [reply app/api/tests/data/example.csv:3]: This now works and I added this specific to the testcase.

**Pull request merged**